composeApp/src/androidMain/kotlin/io/github/couchtracker/ui/components/BackgroundTopAppBar.kt

```diff
@@ -5,40 +5,64 @@ package io.github.couchtracker.ui.components
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.TopAppBarColors
 import androidx.compose.material3.TopAppBarDefaults
 import androidx.compose.material3.TopAppBarScrollBehavior
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.draw.blur
+import androidx.compose.ui.graphics.BlurEffect
 import androidx.compose.ui.graphics.Brush
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.TileMode
+import androidx.compose.ui.graphics.graphicsLayer
 import androidx.compose.ui.unit.dp
-import androidx.compose.ui.unit.times
+import kotlin.math.pow
+
+private const val GRADIENT_STEPS = 16
+private const val BACKGROUND_START_ALPHA = 0.4f
+
+private fun createGradientBrush(color: Color): Brush {
```

Comment:
Can you move this private function at the bottom? We should keep the main Composable function at the top (after the constants), similarly to what we do to classes.